fuzzing/fuzzer_apdu_parser.c

```diff
@@ -0,0 +1,23 @@
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+
+#include "parser.h"
+#include "base58.h"
+#include "bip32.h"
+#include "qrcodegen.h"
+#include "os_io.h"
```

Comment:
```suggestion
#define IO_APDU_BUFFER_SIZE (5 + 255)
```
and delete the `os_io.h` file?

Reply:
Yes, I removed unused headers and directly set `IO_APDU_BUFFER_SIZE` in `fuzzer_apdu_parser.c`.